src/javascript/expression.rs

```diff
@@ -144,6 +146,31 @@ impl<'module> Generator<'module> {
         })
     }
 
+    fn call<'a>(&mut self, fun: &'a TypedExpr, arguments: &'a [CallArg<TypedExpr>]) -> Output<'a> {
+        let fun = self.not_in_tail_position(|gen| gen.expression(fun))?;
+        let arguments = self.not_in_tail_position(|gen| {
+            call_arguments(
+                arguments
+                    .iter()
+                    .map(|element| gen.wrap_expression(&element.value)),
+            )
+        })?;
+        Ok(docvec![fun, arguments])
+    }
+
+    fn fun<'a>(&mut self, arguments: &'a [TypedArg], body: &'a TypedExpr) -> Output<'a> {
+        let tail = self.tail_position;
+        self.tail_position = true;
+        let result = self.expression(body);
+        self.tail_position = tail;
+        Ok(docvec!(
+            docvec!(fun_args(arguments), " => {", break_("", " "), result?)
+                .nest(INDENT)
+                .group(),
+            break_("", " "),
```

Comment:
I think this break should be inside the group` so that the `}` is rendered on its own line

Reply:
:thinking: